### PR TITLE
Adding default Resolution values from ILLIXR

### PR DESCRIPTION
**Pull request by DhairyaBahl**
@e3m3 @mhuzai Kindly review this PR and tell me if I have to correct something or make some changes. Thank You

### CODE REVIEW

File: C/source/generateHologram.cu [hunk 1]
```diff
@@ -25,6 +25,8 @@
 //#define M_CORE_DEBUG
 
 #include "hologram.h"
+#include "global_module_defs.hpp"
```
Comment (e3m3) [inline C/source/generateHologram.cu]: Should be `"common/global_module_defs.hpp"`

File: C/source/generateHologram.cu [hunk 2]
```diff
@@ -37,8 +39,18 @@
...
+#else
+	#define SLM_HEIGHT 1440	
+#endif	
+
+#ifdef DEFAULT_WIDTH
+	#define SLM_WIDTH DEFAULT_WIDTH		//Declaring SLM_WIDTH as DEFAULT_WIDTH from global_module_defs.hpp
+#else	
+	#define SLM_WIDTH 2560
+#endif
+
```
Comment (e3m3) [inline C/source/generateHologram.cu]: Change the names `DEFAULT_HEIGHT` and `DEFAULT_WIDTH` to match the names used to address @mhuzai 's comment in PR [#162](https://github.com/ILLIXR/ILLIXR/pull/162)

**Review by e3m3 — CHANGES_REQUESTED** [review]
Update your [PR](https://github.com/ILLIXR/ILLIXR/pull/162) and have this one match your changes

File: C/source/generateHologram.cu (hunk 2 above)
Comment (DhairyaBahl) [reply C/source/generateHologram.cu]: @e3m3 I don't understand, but i guess #define should not but used here but it should be like this
```
#ifdef DEFAULT_WIDTH
     SML_WIDTH=DEFAULT_WIDTH
#else
```
If yes, then allow me to make these changes. Thanks :)

File: C/source/generateHologram.cu (hunk 1 above)
Comment (e3m3) [inline C/source/generateHologram.cu]:
```suggestion
#include "common/global_module_defs.hpp"
```

File: C/source/generateHologram.cu (hunk 2 above)
Comment (e3m3) [inline C/source/generateHologram.cu]:
```suggestion
// Declaring SLM_WIDTH using global_module_defs.hpp
#ifdef FB_WIDTH
#define SLM_WIDTH ILLIXR::FB_WIDTH
#else
#define SLM_WIDTH 2560
#endif // SLM_WIDTH

// Declaring SLM_HEIGHT using global_module_defs.hpp
#ifdef FB_HEIGHT
#define SLM_HEIGHT ILLIXR::FB_HEIGHT
#else
#define SLM_HEIGHT 1440
#endif // SLM_HEIGHT
```

**Review by e3m3 — CHANGES_REQUESTED** [review]
Here are the matching changes for PRs #162 and #198 from ILLIXR/ILLIXR

**Review by e3m3 — APPROVED** [review]

**Review by mhuzai — APPROVED** [review]

**Pull request merged**